add logic to adjust country side bar for showing projects, members text

smit1678 opened the pull request:
Initial structure and take for how we can adjust the sidebar text for country pages. This will be dependent on if the country has projects and/or members. This is more of a temporary layout until we revamp the project and country pages based on our discussion in https://github.com/hotosm/hotosm-website/issues/128#issuecomment-370768624

### Questions :question: 

Do we want to improve the language more?
I've added some connection to local community links but ideally we pull in https://github.com/osmlab/osm-community-index in the future

## Country with projects, members, and an office
![fireshot capture 14 - humanitarian openstreetmap team_ - http___localhost_4000_where-we-work_uganda_](https://user-images.githubusercontent.com/796838/39867349-6e344358-544c-11e8-99f2-2b511695c2fb.png)

## Country with members only 
![fireshot capture 17 - humanitarian openstreetmap te_ - http___localhost_4000_where-we-work_portugal_](https://user-images.githubusercontent.com/796838/39867347-6df55bc0-544c-11e8-955f-0cf05ec9886a.png)

## Country with out projects or members
![fireshot capture 15 - humanitarian openstreetmap team_ - http___localhost_4000_where-we-work_belize_](https://user-images.githubusercontent.com/796838/39867348-6e14264a-544c-11e8-83d3-74a30f71df27.png)

smit1678 committed:
add logic to adjust country side bar for showing projects, members text

TylerRadford commented:
this looks great @smit1678 
I would just adjust the text slightly to replace "Interested in starting a project"
If project exists display 'Travelling to [country] or want to get in touch about a collaboration?' (plus the contact info you already have)
else 'Have a project or collaboration idea for [country]?' Contact info@hotosm.org

smit1678 committed:
update language to country sidebar, add logic for number of countries shown on project page

smit1678 commented:
Updated the language on country pages. 

I also took the liberty and added some logic to limit the total number of countries shown on the project page:

![fireshot capture 18 - humanitarian openstreetmap team i our work - http___localhost_4000_our-work](https://user-images.githubusercontent.com/796838/39919623-1af0cac0-550c-11e8-93b4-64963b256942.png)
